benchmark/benchmark_adapter.cpp

```diff
@@ -93,11 +100,11 @@ namespace xt
         V a({1, 2, 3, 4});
         V b({1, 2, 3, 4});
 
-        for (auto _ : state)
+        for(auto _ : state)
         {
             V result({0, 0, 0, 0});
             auto n = std::distance(a.begin(), a.end());
-            for (std::size_t i = 0; i < n; ++i)
+            for(std::size_t i = 0; i < n; ++i)
```

Comment:
```suggestion
            for (std::size_t i = 0; i < n; ++i)
```
I know that @JohanMabille requested this, but I find this way too many grouped characters whose readability relies fully on syntax highlighting. And so, I would argue strongly against the no space. 

To settle the debate we could for now also use the variant that is most commonly used in xtensor

Reply:
Let me add an argument, a space is used also here :  
* https://en.cppreference.com/w/cpp/language/for 
* https://learn.microsoft.com/en-us/cpp/cpp/for-statement-cpp?view=msvc-170